core/spec/models/spree/concerns/user_address_book_spec.rb

```diff
@@ -167,9 +239,23 @@ module Spree
           expect(subject).to eq address
         end
 
-        it "sets it as default" do
-          subject
-          expect(user.default_address).to eq address
+        context "sets it as default" do
+          context "shipping address" do
+            it do
```

Comment:
`it` blocks should always have a description unless you're using one-liners, so that it's clear what you're testing in the example.

Reply:
Done (long ago)